sensor_msgs/CompressedImage image
---
amr_nn_controller_service/Command2D commands

